setenv.sh

```diff
@@ -1,46 +1,35 @@
 #!/bin/sh
-#set -x
+#
+# Set up environment variables for general build tool to operate
+#
 if ! [ -f ./setenv.sh ]; then
-	echo "Need to source from the setenv.sh directory" >&2
-else
-	export _BPXK_AUTOCVT="ON"
-	export _CEE_RUNOPTS="FILETAG(AUTOCVT,AUTOTAG),POSIX(ON),TERMTHDACT(MSG)"
-	export _TAG_REDIR_ERR="txt"
-	export _TAG_REDIR_IN="txt"
-	export _TAG_REDIR_OUT="txt"
-
-	if [ -z "$GIT_ROOT" ]; then
-		export GIT_ROOT=/rsusr/ported/bin
-	fi  
-
-	export PATH=$PWD/bin:$PATH
-
-	# See perlbuild.sh for valid values of PERL_xxx variables
-  if [ -z "${PERL_VRM}" ]; then
-	  export PERL_VRM="blead" #maint-5.34|blead
-  fi
-
-  if [ -z "${PERL_OS390_TGT_AMODE}" ]; then
-	  export PERL_OS390_TGT_AMODE="64" # 31|64
-  fi
-
-  if [ -z "${PERL_OS390_TGT_LINK}" ]; then
-	  export PERL_OS390_TGT_LINK="dynamic" # static|dynamic
-  fi
+  echo "Need to source from the setenv.sh directory" >&2
+  return 0
+fi
 
-  if [ -z "${PERL_OS390_TGT_CODEPAGE}" ]; then
-	  export PERL_OS390_TGT_CODEPAGE="ascii" # ebcdic|ascii
-  fi
+export PORT_ROOT="${PWD}"
+export PORT_TYPE="GIT"
 
-  if [ -z "${PERL_OS390_TGT_LOG_DIR}" ]; then
-	  export PERL_OS390_TGT_LOG_DIR=${PWD} 
-  fi
+PORT_GIT_DIR=perl
 
-	export PERL_ROOT="${PWD}"
+if [ ! -z "${PERL_INSTALL_DIR}" ]; then
+  install_dir=${PERL_INSTALL_DIR}
+else
+  install_dir="${HOME}/local/perl"
```

Comment:
is this temporary? If it needs to be permanent, maybe we need to change our approach for 'install' ?